docker-compose.yml

```diff
@@ -2,18 +2,11 @@
 version: '3'
 services:
   web:
-    build: .
+    build: 
+      context: .
+      dockerfile: development.Dockerfile
     volumes:
-      # Mount specific sub-directories into image for local development
-      - ./app:/upaya/app
-      - ./certs:/upaya/certs
-      - ./config:/upaya/config
-      - ./db:/upaya/db
-      - ./keys:/upaya/keys
-      - ./lib:/upaya/lib
-      - ./log:/upaya/log
-      - ./pwned_passwords:/upaya/pwned_passwords
-      - ./vendor:/upaya/vendor
+       - .:/upaya
```

Comment:
I switched this back to mount the whole dir... it's just easier in development, and we don't have the same worries as we do in prod...

Reply:
This causes issues if the user does not already have npm_modules in their local identity-idp directory.  HOWEVER... I think I have a solution, push this now and we can revisit later.